spec/lib/diaspora/markdownify_email_spec.rb

```diff
@@ -8,12 +8,12 @@
 
     it 'should autolink a hashtag' do
       markdownified = @html.preprocess("#tag")
-      expect(markdownified).to eq("[#tag](http://localhost:9887/tags/tag)")
+      expect(markdownified).to eq("[#tag](#{AppConfig.url_to('tags/tag')})")
```

Comment:
It gives me
```
     NoMethodError:
       undefined method `tag_path' for #<RSpec::ExampleGroups::DiasporaMarkdownifyEmail::Preprocess:0x00000009d073a8>
```
for that file.

I suppose that happens because it is not a controller and not the Rails application context.